torch/testing/_internal/common_utils.py

```diff
@@ -314,6 +314,21 @@ def run_tests(argv=UNITTEST_ARGS):
 IS_MACOS = sys.platform == "darwin"
 IS_PPC = platform.machine() == "ppc64le"
 
+# Intel Cascade Lake has CPU Family 6, Model 85, and Stepping between 5 to 7
+def is_cascade_lake():
```

Comment:
IMO, it's better to check for the flag avx512_vnni in "Flags" instead of the specific cpu_family/cpu_model as generations after cascade_lake (e.g., cooper lake) will also have avx512_vnni.

Reply:
Thanks for taking a look! I had written this before I discovered the root-cause, and forgot to update it.
I'll change it. Hopefully, this test would be fixed & this skip won't be required.

Reply:
BTW, @dskhudia, is there a quantization you'd recommend to gauge speedup? Thanks!

Reply:
Do you mean any quantization operator?

Reply:
Sorry, there was a typo! I had omitted `benchmark` by mistake. Even if multiple benchmarks exist to test the performance of different quantization operators/kernels, I can try running them. Thanks!

Reply:
You can run any (or all) of functions from the following file as these were implemented with avx2: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

You just need to find the corresponding operator. For example, to run qlamp_kernel (https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp#L617) look for corresponding dispatch qclamp_stub (https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp#L3112) and then the actual operator should be in one of the files in this dir: aten/src/ATen/native/quantized/cpu .

For qclamp, aten operator is quantized::clamp (https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/qclamp.cpp#L163)

You can write benchmarks around these operators. Some existing benchmarks are here: benchmarks/operator_benchmark/pt . For example, qcat in https://github.com/pytorch/pytorch/blob/master/benchmarks/operator_benchmark/pt/qcat_test.py

Reply:
Thanks a lot for your advice, @dskhudia!